### ISSUE TITLE
Scroll bar missing

### ISSUE THREAD

**Issue by mueller-ma** [axxapy/apkExtractor#12]
With the latest version 1.3 I cannot see any scroll bar on the right edge.
Android 7.1.2
Lineageos 14.1

**Comment by axxapy** [comment]
Can you scroll using touch gestures? By swiping up or down?

**Comment by mueller-ma** [comment]
Yes, but I dont see an indicator at which position I am in the list view.
